### PR TITLE
feat: Add more special tokens

### PR DESCRIPTION
**Pull request by tomeras91**
feat: Add eos_id and newline_id

### CODE REVIEW

File: ai21_tokenizer/jurassic_tokenizer.py [hunk 1]
```diff
@@ -30,8 +30,11 @@ def __init__(
         self.unk_id = config.get("unk_id")
         self.eop_id = config.get("eop_id")
         self.bos_id = config.get("bos_id")
+        self.eos_id = config.get("eos_id")
 
         self._newline_piece = config.get("newline_piece")
+        self.newline_id = self.convert_tokens_to_ids(self.newline_piece)
```
Comment (asafgardin) [inline ai21_tokenizer/jurassic_tokenizer.py:36]: Access protected attribute instead

File: ai21_tokenizer/jurassic_tokenizer.py [hunk 2]
```diff
@@ -30,8 +30,11 @@ def __init__(
         self.unk_id = config.get("unk_id")
         self.eop_id = config.get("eop_id")
         self.bos_id = config.get("bos_id")
+        self.eos_id = config.get("eos_id")
 
         self._newline_piece = config.get("newline_piece")
+        self.newline_id = self.convert_tokens_to_ids(self._newline_piece)
```
Comment (asafgardin) [inline ai21_tokenizer/jurassic_tokenizer.py:36]: @tomeras91 I see that on line 48 we already have a `newline_id` whats the difference?

Comment (tomeras91) [reply ai21_tokenizer/jurassic_tokenizer.py:36]: You're right. I missed it. I'll just make it "public"

**Review by etang-ai21 — APPROVED** [review]

File: ai21_tokenizer/jurassic_tokenizer.py (hunk 2 above)
Comment (asafgardin) [reply ai21_tokenizer/jurassic_tokenizer.py]: Great. Also notice that another function `encode` is using it, so make sure you rename it in all places

**Pull request merged**